products/pubsub/terraform.yaml

```diff
@@ -33,7 +33,6 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       - !ruby/object:Provider::Terraform::Examples
         name: "pubsub_topic_geo_restricted"
         primary_resource_id: "example"
-        primary_resource_name: "fmt.Sprintf(\"example-topic-%s\", context[\"random_suffix\"])"
```

Comment:
What's different about this value vs the one above that we remove one and modify the other?

Reply:
IAM resource generation only needs one example to have a `primary_resource_name` so that it can construct the import ID for the generated tests. This doesn't need to exist, was probably copy/pasted